src/components/shared/FundsRaisedCounter.tsx

```diff
@@ -27,7 +27,8 @@ function FundsRaisedCounter() {
   );
   const accountTokenSnapshot = data?.items[0];
   const amount = useFlowingAmount(
-    BigInt(accountTokenSnapshot?.totalAmountStreamedInUntilUpdatedAt ?? 0) +
```

Comment:
@tnrdd Was this change intentional? I think it takes us back to understating the amount raised.

Reply:
Using only `totalAmountStreamedUntilUpdatedAt` is not enough because it wouldn't count the 0.005 received on parcels claim, `balanceUntilUpdatedAt + totalAmountStreamedOutUntilUpdatedAt + totalAmountTransferredUntilUpdateAt` should be the correct value.

Reply:
Got it. On the same page now. TY!